src/app/states/db/page.mdx

```diff
@@ -0,0 +1,51 @@
+export const metadata = {
+  title: 'db',
+  description: 'An app state of database synchronization. in Inkdrop. This section explains how to fetch and manipulate the `notes` state.',
+}
+
+# books
+
+An [app state](/manual/flux-architecture) of database synchronization.
+You can obtain the `notes` state and integrate it with React components.{{ className: 'lead' }}
+
+
+---
+
+## Data Structure
+
+<Row>
+  <Col>
+    <Properties>
+      <Property name="isSyncing" type="?boolean">
+        true if the database is syncing.
+      </Property>
+      <Property name="lastSyncTime" type="?number">
```

Comment:
Since Flow (FlowType) is no longer up to date, and as far as I know Inkdrop is migrating to Typescript, the Typescript syntax should also be used for describing types.
```suggestion
      <Property name="isSyncing" type="boolean | undefined">
        `true` if the database is syncing.
        `undefined` if it has not ever been synced yet since the app launched.
      </Property>
      <Property name="lastSyncTime" type="number | undefined">
```

Reply:
right, the typescript definition currently looks like so:

```ts
export type DBSyncState = {
  lastSyncStatus?: string
  isSyncing?: boolean
  lastSyncTime: number | null | undefined
  lastSyncError?: Error | null
}
export type DBChanges = {
  [prop: string]: any
  cancel: (...args: Array<any>) => any
}
export type DBState = DBSyncState & {
  changes?: DBChanges
}
export const initialDBState: DBState = {
  lastSyncTime: null
}
```